packages/library-legacy/src/programs/vote.ts

```diff
@@ -96,6 +97,17 @@ export type UpdateValidatorIdentityParams = {
   nodePubkey: PublicKey;
 };
 
+export type CompactUpdateVoteStateParams = {
+  voteAccount: PublicKey,
```

Comment:
nit: Just to follow the pattern of the others:

```suggestion
  votePubkey: PublicKey,
```